lib/parser/parse-tree.h

```diff
@@ -3529,6 +3529,12 @@ struct OpenMPDeclareTargetConstruct {
   std::variant<WithClause, WithExtendedList, Implicit> u;
 };
 
+struct OpenMPDeclareTargetConstruct {
+  TUPLE_CLASS_BOILERPLATE(OpenMPDeclareTargetConstruct);
+  CharBlock source;
+  std::tuple<Verbatim, OpenMPDeclareTargetSpecifier> t;
```

Comment:
What is the difference between the source range in `source` and the one in `Verbatim`?

Reply:
For example, `!$omp declare target to(Q)`. The `Verbatim` is the directive `declare target` itself; the `source` is `declare target to(Q)`.